polling_stations/templates/fragments/cancelled_election.html

```diff
@@ -1,32 +1,22 @@
 {% load i18n_with_welsh %}
-{% load markdown_filter %}
 {% if cancelled_election.metadata.cancelled_election.title %}
     <h2>{{ cancelled_election.metadata.cancelled_election.title }}</h2>
 {% else %}
     <h2>{% trans "Cancelled Election" %}</h2>
```

Comment:
If we are here, we know that the election isn't going to take place and we know it hasn't been rescheduled _yet_ (i.e: there is no replacement ballot attached yet), but we don't actually know if it is postponed or cancelled.

I've changed the inline text to "..will not go ahead as planned".
In terms of the heading here, we could also change it to
"This election will not go ahead as planned"
which is a little wordy but gets rid of cancelled. if you've got any other ideas for generic words, I'm open to ideas.

This will be shown more frequently than the corresponding version on WCIVF